lib/Runtime/Library/JavascriptArray.cpp

```diff
@@ -1109,15 +1109,8 @@ namespace Js
 
         if (callInfo.Count < 2)
         {
-            if (pNew == nullptr)
```

Comment:
pNew can never be nullpre. If you read the code above, we declare it and start checking if it is nullptr. I think there was legitimate case where pNew was non-nullptr, and during refactoring, we removed that code. The code that have removed is dead code that can't happen atleast in current state of code.

---
In reply to: [95442434](https://github.com/Microsoft/ChakraCore/pull/2325#discussion_r95442434) [](ancestors = 95442434)